2/contrib/jenkins/install-jenkins-core-plugins.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash -eu
+
+source /usr/local/bin/install-plugins-internals.sh
```

Comment:
part of the solution to your issue might be to not source this unless you're in the rpms=false case.

Reply:
I was about to suggest that ... we still want folks to be able to call inner.sh directly .. i.e. install-plugins.sh directly ... as they may have been doing that in their extension scenarios

Reply:
assuming I can source within the logic like that and pass the parameters

Reply:
https://stackoverflow.com/questions/39768712/how-to-pass-arguments-to-a-script-invoked-by-source-command

Reply:
or you can just call the script rather than source it.

Reply:
the latter is working ... will push that shortly